.editorconfig

```diff
@@ -0,0 +1,12 @@
+# To use .editorconfig you need to install a plugin from
```

Comment:
Frontend tests are failing. I'm stop reviewing at this point.

Reply:
Updated failed snapshot

Reply:
One sure, the other is still wrong. Please make sure tests are green before putting this into review.

Reply:
Updated tests to reflect config file with empty object

Reply:
@DannyShopgate what you did in last commit is using empty config indeed but now you just also updated the snapshot so it tests against component that returns null.

We talked about it and I told you you need to mock the config in order to test if it renderers. Testing component that doesn't render anything doesn't make much sense.

Please adjust this test.

Reply:
Heyo @sznowicki I believe this is the test in questions: 
```javascript
const noManualUrls = null;
const manualUrls = [{
  label: 'mock',
  url: 'mock',
  type: 'mock',
}];
const mockConfig = { sectionTitle: 'mock' };
describe('<ManualsSection />', () => {
  it('should not render if no manual urls', () => {
    const wrapper = mount((
      <ManualsSection config={mockConfig} manualUrls={noManualUrls} />
    ));
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('div').exists()).toBe(false);
  });
  it('should render if provided with manual urls', () => {
    const wrapper = mount((
      <ManualsSection config={mockConfig} manualUrls={manualUrls} />
    ));
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('div').exists()).toBe(true);
  });
});
```
I i did mock a config passed to the component. The second test, tests if the component does render.

Reply:
No, we're still talking about the ProductDescriptionAfter, the one that was failing before because of missing config mock.

Not it still tests against "live" config which is an empty file atm. It must be mocked, otherwise the test renders empty component, which doesn't make any sense for testing.

Reply:
Right, of course! Updated